# CHEM 223 - Organic Chemistry 3 Lab


## Foreword

This notebook contains calculations for my third and final quarter of organic chemistry lab. At the time of writing, it's April 2020 and deep in quarantine, so this lab is online. I know what you're thinking, an online ochem lab? My days of broken glassware, fudged data, and poorly taken IR and proton NMR spectra are over. For the past two quarters, organic chemistry labs have been a welcome relief from the rest of my reading-on-a-computer typing-on-a-computer staring-at-a-computer coursework. This was where the mild pain of organic chemistry was supposed to come together in a tangible and rewarding way. Guess not. I won't miss stressing about finishing reactions and cleaning up under time pressure, though.

In [19]:
class Reagent:
    def __init__(self, name, grams=None, moles=None, mmoles=None, mL=None, mw=None, density=None, M=None, pKa=None): # g/mL
        self.name = name
        self.grams = grams
        self.moles = moles
        self.mmoles = mmoles
        self.mL = mL
        self.mw = mw
        self.density = density
        self.M = M
        self.pKa = pKa
        
        for i in range(20): # for dependent measurements
            self.grams = self._calc_grams()
            self.moles = self._calc_moles()
            self.mmoles = self._calc_mmoles()
            self.mL = self._calc_mL()
            
    
    def _calc_grams(self):
        if self.grams:
            return float(self.grams)
        elif self.moles and self.mw:
            return float(self.moles) * float(self.mw)
        elif self.mmoles and self.mw:
            return (float(self.mmoles) / 1000.0) * float(self.mw)
        elif self.mL and self.density:
            return float(self.mL) * float(self.density)
        return None
    
    def _calc_moles(self):
        if self.moles:
            return float(self.moles)
        elif self.mmoles:
            return float(self.mmoles) / 1000.0
        elif self.grams and self.mw:
            return float(self.grams) / float(self.mw)
        elif self.mL and self.M:
            return float(self.mL) / 1000.0 * self.M
        return None
    
    def _calc_mmoles(self):
        if self.mmoles:
            return float(self.mmoles)
        elif self._calc_moles():
            return self._calc_moles() * 1000.0
        return None
    
    def _calc_mL(self):
        if self.mL:
            return self.mL
        elif self.grams and self.density:
            return float(self.grams) / float(self.density)
        elif self.moles and self.mw and self.density:
            return float(self.moles) * float(self.mw) / float(self.d)
        elif self.mmoles and self.mw and self.density:
            return float(self.mmoles) * float(self.mw) / float(self.d) / 1000.0
        return None
    
    def __str__(self):
        s = self.name
        if self.grams:
            s += f" {self.grams} g"
        if self.moles:
            s += f" {self.moles} mol"
        if self.mmoles:
            s += f" {self.mmoles} mmol"
        return s
    
#     def percent_yield()

## EXPERIMENT 1: SYNTHESIS OF METHYL SALICYLATE: A FISCHER ESTERIFICATION

### Reagents

In [116]:
sa = Reagent("salicylic acid", grams=9.7, mw=138.121)
meoh = Reagent("MeOH", mL=25.0, mw=32.04, density=0.792, pKa=15.54)
h2so4 = Reagent("Conc. H2SO4", mL=4.0, M=18.0, pKa=-2.8)
quiz_incorrect_h2so4 = Reagent("Incorrect amount pure H2SO4", mL=1.0, density=1.83, mw=98.07, pKa=-2.8)

### Equivalents

In [117]:
import pandas as pd

reagents = [sa, meoh, h2so4, quiz_incorrect_h2so4]
mmoles = [[r.name, r.mmoles] for r in reagents]
df = pd.DataFrame(mmoles, columns=["name", "equivalents"])
df['equivalents'] = df['equivalents'] / sa.mmoles
df

name  equivalents
0               salicylic acid     1.000000
1                         MeOH     8.799554
2                  Conc. H2SO4     1.025228
3  Incorrect amount pure H2SO4     0.265707

### Theoretical Yield

1:1 molar ratio between salicylic acid reagent and methyl salicate product

In [109]:
ms = Reagent("methyl salicylate", moles=sa.moles, mw=152.1494)
print(f"{ms.grams:.2f} g methyl salicylate")

10.69 g methyl salicylate


### Percent yield

In [119]:
py = 7.4 / ms.grams * 100
print(f"percent yield: {py:.2f}. Nice.")

percent yield: 69.25. Nice.


### Properties and uses of salicylic acid and methyl salicylate

Salicylic acid is a whitish-tan solid at room temperature. It can be used to remove the outer layer skin and is often found in skincare and acne products. Salicylic acid is also an active metabolite of the drug aspirin and is responsible for the anti inflammatory properties of aspirin.

Methyl salicylate, aka wintergreen oil, is a colorless liquid at room temperature with a minty or fruity odor. It is commonly used in fragrances, foods, and beverages. It is also used in traditional medicine.


### Procedure in lab manual vs youtube video

[youtube video](https://www.youtube.com/watch?v=lJLP2bcXDqY)

The lab manual procedure used different and smaller quantities of reagents. No boiling stones in the video. Reaction mixture in the video was refluxed for twice as long. The video used a rotovap immediately after refluxing to remove some of the excess methanol. In the video, only cold water was added to the sep funnel. The lab manual calls for cold water, EtOAc, and brine to be added to the sep funnel. The video showed an extra final wash step. When the rest of the extraction was completed, the lab manual procedure called for rotovapping off the EtOAc. The video mentioned but did not show the use of MgSO4 to remove excess water from the product.


### Percent yield

69.25%


### Rf/Polarity of methyl salicylate vs salicylic acid

Salicylic acid would have a lower Rf value because the carboxylic acid group in salicylic acid is more polar than the ester group in methyl salicylate.


### IR Spectrum of methyl salicylate vs 

Methyl salicylate IR spectrum has a peak at 1650 cm-1 from ester C=O stretch. There are also C-O stretches in the fingerprint region from 1000 - 1300 cm-1 characteristic of an ester, which are absent in the IR spectra of salicylic acid. The IR spectra of both methyl salicylate and salicylic have O-H stretches around 3000 to 3500 cm-1.


### HNMR of methyl salicylate

The signal at 10.5 is from the phenol H. This is not a good signal to base the integrations off because it is broad and difficult to integrate accurately. This means the other integrations are not directly indicitave of the number of protons causing the signals. The 4 aromatic protons cause the peaks between 6.7 and 7.9, but overlap and are difficult to distinguish from each other. The singlet at 3.9 is caused by the 3 protons in the methyl group on the end of the ester. It would be better to integrate the signals based off of this signal, using a value of 3, because it is strong and distinct.


### 13C NMR of methyl salicylate

The C in the ester is the furthest signal downfield. The second furthest downfield signal is likely the C attached to the OH.


### Most interesting thing I learned

Since salicylic acid is the principle metabolite of aspirin, I enjoyed reading about how aspirin is absorbed by the body and metabolized into salicylic acid.

## EXPERIMENT 2: OXIDATION OF BORNEOL WITH NaOCl

In [5]:
borneol = Reagent("borneol", grams=2.0, mw=154.25)
borneol.mmoles

12.965964343598054

In [14]:
actual_borneol = Reagent("borneol", grams=2.03, mw=154.25)
camphor = Reagent("camphor", moles=actual_borneol.moles, mw=152.23)
pyield = 1.54 / camphor.grams * 100
print(f"Percent yield: {pyield:.1f}%")

Percent yield: 76.9%


### Report Questions


### 4. What would the key differences be between the IR of starting material and the product?

Starting material would have an alcohol O-H stretch around 3000-3500 from the alcohol, C-O stretches in the fingerprint region. Product would have a ketone C=O stretch around 1700.


### 5. List three pertinent principles of green chemistry that were followed in the procedure, briefly describe how they were applied to our experiment. You can answer this in comparison to using  Jones reagent for oxidizing an alcohol to a ketone

3: Design less hazardous chemical syntheses. Less toxic household bleach was used to oxidize an alcohol instead of toxic chromium reagents.

8: Avoid chemical derivatives (and protecting or blocking groups). The synthesis did not involve any protecting groups and the alcohol in borneol was oxidized directly to a ketone.

11: Analyze in real time. The temperature of the reaction was monitored to determine the progress. The reaction mixture was tested with a starch iodine test strip to verify that enought oxidizing agent was present to oxidize all of the reagents. This helped confirm that reagents were not left unreacted and wasted.


### 6. The proton NMR of camphor can be found on the home page. Proton C has been labeled. Why is the proton NMR so complicated? Why is the N+1 rule for multiplicity not followed?

Protons C and D are not equivalent. Proton C is split by both protons E and D. The doublet of triplets results when each peak of the doublet from splitting by proton E is split into a triplet by proton D.

# EXPERIMENT 3: PREPARATION OF DIBENZALACETONE

In [14]:
benzaldehyde = Reagent("Benzaldehyde", mw=106.124, density=1.04, mmoles=30.0)
print("Benzaldehyde")
print(benzaldehyde.grams, "g")
print(benzaldehyde.mL, "mL")

acetone = Reagent("Acetone", mw=58.08, density=0.784, mmoles=15.0)
print("Acetone")
print(acetone.grams, "g")
print(acetone.mL, "mL")

Benzaldehyde
3.1837199999999997 g
3.06126923076923 mL
Acetone
0.8712 g
1.1112244897959183 mL


Why does the carbon of the enolate anion attack the carbonyl of the benzaldehyde? What doesn't  the oxygen on the enolate attack?  Below are two computer calculations of the a) the electrostatic map of the enolate anion of acetone  b) the calculated HOMO of the enolate ion of acetone. What do these pictures tell you about the reactivity of enolate? 

Oxygen attracts the electrons more strongly and is less willing to share them. The HOMO of the enolate ion shows that the electrons spend more time around the carbon, not the oxygen, which makes it more likely that the electrons would attack when they are near the carbon.

Why does the enolate ion of acetone attack the electrophilic carbonyl of the aldehdye and not another acetone molecule?

The carbon of the the aldehyde is more electrophilic because it is next to a benzene ring.

In [3]:
# percent yield
benzaldehyde = Reagent("Benzaldehyde", mw=106.124, density=1.04, mL=2.0)
dibenzylacetone = Reagent("dibenzylacetone", moles=benzaldehyde.moles / 2.0, mw=234.29)
print(f"Percent yield: {(2.15 / dibenzylacetone.grams) * 100:.2f}%")

Percent yield: 93.64%


In [7]:
# coupling constants
print(f"{(7.1360 - 7.0962) * 300 :.2f}")

11.94


4. Describe key differences in the IR of the starting materials and the product that would confirm your reaction was successful 

The conjugated aldehyde in the starting material appears at a higher frequency than the conjugated ketone C=O stretch in the product. The aldehyde C-H stretch around 2700 cm-1 is also not present in the dibenzylacetone product.


5. Dibenzylacetone is an active ingredient in sunscreen. How does it protect your skin from the sun, structurally what is responsible  this property?

Dibenzylacetone absorbs light in the IR spectrum. This is because it is a highly conjugated system.

6. Watch this  video on Flow chemistry https://www.youtube.com/watch?v=bPmOsSswppI

This cute video outlines some of  the advantages of flow chemistry. Do your own internet research:

 

a)What are some of the challenges/disadvantages of flow chemistry?

 

b) Search and find 1 compound that has been synthesized using flow chemistry either in academia or in industry. Find the original journal article, provide the names of the compound, the title of the paper and cite the article.  Be sure to read the document under experiment 3 that tells you how to access articles remotely and  the format they should be cited in.  

c) Explain why the researchers chose to use flow chemistry for that compound. 


Advantages: high temperature, increased safety, fast mixing of compounds, easier to automate, coupling purification with the reaction, easier to scale up

Disadvantages: Specialized equipment is required, on a small scale the surface to volume ratios are good but this does not always apply to scaling up.

Baxendale, Ian R.; Jon Deeley; Charlotte M. Griffiths-Jones; Steven V. Ley; Steen Saaby; Geoffrey K. Tranmer. Chemical Communications 2006. 0 (24): 2566–2568.

In the paper "A flow process for the multi-step synthesis of the alkaloid natural product oxomaritidine: a new paradigm for molecular assembly", the authors synthesized oxomaritidine using flow chemistry. They chose flow chemistry because "no isolation at intermediate stages was necessary, no use was made of labour intensive techniques such as product distillation or crystallisation, nor was column chromatography required to furnish materials of appropriate quality in order to progress to the next step in the synthesis." The result was lots of saved time and labor.

# EXPERIMENT 4: MULTI-STEP AROMATIC SYNTHESIS

In [17]:
fmap = Reagent("4-methoxyacetophenone", mw=150.17, grams=5.0)
conc_h2so4 = Reagent("conc. H2SO4", M=18.0, mL=20.0)
conc_hno3 = Reagent("conc. HNO3", M=16.0, mL=2.5)

print(fmap)
print(conc_h2so4)
print(conc_hno3)

4-methoxyacetophenone 5.0 g 0.03329559832190185 mol 33.29559832190185 mmol
conc. H2SO4 0.36 mol 360.0 mmol
conc. HNO3 0.04 mol 40.0 mmol


In [21]:
# theoretical yield of 3-Nitro-4-methoxyacetophenone
tnfmap = Reagent("3-nitro-4-methoxyacetophenone", mw=195.17, moles=fmap.moles)
print(tnfmap)

3-nitro-4-methoxyacetophenone 6.498301924485583 g 0.03329559832190185 mol 33.29559832190185 mmol


In [22]:
step2_tnfmap = Reagent(tnfmap.name, mw=tnfmap.mw, grams=3.3)
print(step2_tnfmap)`

3-nitro-4-methoxyacetophenone 3.3 g 0.016908336322180664 mol 16.908336322180663 mmol


In [24]:
q_fmap = Reagent("4-methoxyacetophenone", mw=150.17, grams=3.0)
q_tnfmap = Reagent("3-nitro-4-methoxyacetophenone", mw=195.17, moles=q_fmap.moles)
print(q_tnfmap)

3-nitro-4-methoxyacetophenone 3.89898115469135 g 0.019977358993141108 mol 19.97735899314111 mmol


In [26]:
print(f"{3.0 / q_tnfmap.grams * 100:.2f}%")

76.94%


In [27]:
print(1.00 * 0.94 * 0.86)

0.8083999999999999
